src/VideoJS.as

```diff
@@ -94,39 +94,43 @@ package{
                 }
             }
             finally{}
-            
-            
-            
+
+
+
             setTimeout(finish, 50);
 
         }
-        
+
         private function finish():void{
-            
+
             if(loaderInfo.parameters.mode != undefined){
                 _app.model.mode = loaderInfo.parameters.mode;
             }
-            
-            if(loaderInfo.parameters.eventProxyFunction != undefined){
+
+            // Hard coding these in for now until we can come up with a better solution for 5.0 to avoid XSS.
+            _app.model.jsEventProxyName = 'videojs.Flash.onEvent';
```

Comment:
I also meant whether any changes in the swf are needed where these are used. It sounds like it doesn't.